### PR TITLE
BaSD-MS-138: Sessios Form: date validation on Postulants and admins

### PR DESCRIPTION
**Pull request by nailaghigo**
Postulant and Admin roles:
Add validation to the input date so that the entered date is greater than or equal to today's date.

Notes -> this validation only consider dates, not hours 

This PR need changes at https://github.com/BaSD-2021/Esteban-MindSet-server/pull/70 to run correctly

### CODE REVIEW

File: src/Components/Admin/Sessions/Form/index.js [hunk 1]
```diff
@@ -96,6 +96,22 @@ function sessionsForm() {
...
+
+    const now = new Date();
+    const year = now.getFullYear();
+    const month = now.getMonth() + 1 < 10 ? `0${now.getMonth() + 1}` : now.getMonth() + 1;
+    const date = now.getDate() < 10 ? `0${now.getDate()}` : '';
+
+    let nowValue = Math.round(new Date(`${year}-${month}-${date}`).getTime() / 1000);
+    return dateValue >= nowValue ? undefined : 'Invalid date';
+  };
+
```
Comment (fjmorabito) [inline src/Components/Admin/Sessions/Form/index.js]: This validation should only work when you are creating a new one, not when editing.

File: src/Components/Postulant/Sessions/Form/index.js [hunk 2]
```diff
@@ -96,6 +96,26 @@ function sessionsForm() {
...
+
+    const now = new Date();
+    const year = now.getFullYear();
+    const month = now.getMonth() + 1 < 10 ? `0${now.getMonth() + 1}` : now.getMonth() + 1;
+    const date = now.getDate() < 10 ? `0${now.getDate()}` : '';
+
+    let nowValue = Math.round(new Date(`${year}-${month}-${date}`).getTime() / 1000);
+    return dateValue >= nowValue ? undefined : 'Invalid date';
+  };
+
```
Comment (fjmorabito) [inline src/Components/Postulant/Sessions/Form/index.js]: Same here

**Review by fjmorabito — CHANGES_REQUESTED** [review]

File: src/Components/Postulant/Sessions/Form/index.js (hunk 2 above)
Comment (nailaghigo) [reply src/Components/Postulant/Sessions/Form/index.js]: done

File: src/Components/Admin/Sessions/Form/index.js (hunk 1 above)
Comment (nailaghigo) [reply src/Components/Admin/Sessions/Form/index.js]: done

**Review by veronabaudracco — APPROVED** [review]

**Review by fjmorabito — APPROVED** [review]

**Pull request merged**